services/api/handlers/projects.js

```diff
@@ -0,0 +1,225 @@
+var boom = require('boom');
+
+exports.post = {
+  create: function(request, reply) {
+    request.server.methods.projects.create(
+      [
+        request.params.user,
+        null,
+        request.server.plugins['webmaker-api-service'].version,
+        request.payload.title,
+        JSON.stringify(request.payload.thumbnail)
```

Comment:
we could, but the thumbnail field will always be at least an empty object, since it's got a default setting from the Joi validation